department-of-veterans-affairs/va.gov-cms
# VACMS-1883: Add EWA entity metadata block.

## Description
See #1883 

## Testing done
Visual / Behat

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/91362467-8b041f00-e7c8-11ea-993f-226a29fed21a.png)

## QA steps
As any user, go to the view page for any node, and visually verify that EWA metadata block appears.
